test/config/install_default.json

```diff
@@ -1,6 +1,11 @@
 {
-  "test-config": {
-      "template": "ora-ubuntu-1604.ova",
+  "install-config": {
```

Comment:
No this should not have been changed. Please use "template": "ora-ubuntu-1604.ova" here